### PR TITLE
refactor: ArtworkFilter tracks clicks by default

### PR DESCRIPTION
**Pull request by eessex**
- Removes `ArtworkFilter`'s prop `onArtworkBrickClick` and replaces with `clickedMainArtworkGrid` cohesion event
- Renames `ArtworkFilterArtworkGrid2` to `ArtworkFilterArtworkGrid`
- Updates `commercialFilterParamsChanged` event to use Cohesion
- Removes `2` from `pageType` to ensure that `show2`/`fair2` etc do not error in staging.

Depends on https://github.com/artsy/cohesion/pull/130/files

### CODE REVIEW

File: src/lib/getContextPage.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export function getContextPageFromClient(): {
 
 export const formatOwnerTypes = (path: string) => {
   const type = path.split("/")[1]
-  let formattedType = camelCase(type)
+  let formattedType = camelCase(type).replace("2", "")
```
Comment (damassi) [inline src/lib/getContextPage.ts:56]: Can we add a comment here describing what this does? This change could be unclear in the future

File: src/v2/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx [hunk 2]
```diff
@@ -138,7 +116,7 @@ export const ArtistArtworkFilterRefetchContainer = createRefetchContainer(
           width: $width
         ) {
           id
-          ...ArtworkFilterArtworkGrid2_filtered_artworks
+          ...ArtworkFilterArtworkGrid_filtered_artworks
```
Comment (damassi) [inline src/v2/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx:119]: 🙏

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx [hunk 3]
```diff
@@ -109,7 +109,6 @@ export interface ArtworkFilterContextProps {
   setCounts?: (counts: Counts) => void
 
   // Handlers
-  onArtworkBrickClick?: (artwork: any, props: any) => void
```
Comment (damassi) [inline src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx:112]: Should we be deleting this API feature? It seems handy and for the time being can remain unused

**Review by damassi — APPROVED** [review]
Couple minor comments, otherwise LGTM!

**Review by mzikherman — APPROVED** [review]
So nice! Moving `onArtworkBrickClick` up into the base grid makes a lot of sense.

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx (hunk 3 above)
Comment (eessex) [reply src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx:112]: Looks like we've only ever used it for tracking, so I'm hoping that by removing there is less ambiguity on how to hook this up correctly.

Comment (damassi) [reply src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx:112]: Cool, sounds good 👍

**Pull request merged**